.env

```diff
@@ -20,4 +20,13 @@ REDIS_PORT_6379_TCP_PORT=
 
 SECRET_KEY_BASE=
 SIDEKIQ_CREDS=
-DEFAULT_PROXY_TARGET=
\ No newline at end of file
+DEFAULT_PROXY_TARGET=
+
+# Adobe Campaigns information and credentials
+ADOBE_ORG_ID=asdf
+ADOBE_ORG_NAME=cru
+ADOBE_TECH_ACCT=asdf
+ADOBE_API_KEY=asdf
+ADOBE_API_SECRET=asdf
+ADOBE_SIGNED_JWT=asdf
+ADOBE_SERVICE_NAME=GodToolsWelcomeSeries
```

Comment:
we will need the ability to support multiple different adobe campaign series, so my guess is this probably shouldn't be defined as an environment variable. Instead the `destination` model should track whatever is needed to identify/submit to a specific series.

I don't know the campaigns API, but it might be able to share the API_KEY, API_SECRET, and SIGNED_JWT across all series and it may just be the series/service identifier that needs to be configured per destination.

Reply:
hi @frett the ticket originally said that I should use a specific series. But yes, it shouldn't be a problem to store the series name in a different part, I think the `destinations` table would be the perfect place, so callers could use different `destination_id` depending which series they want.

Reply:
I'll go update the ticket this afternoon to reflect the desire to support multiple campaigns.

Reply:
@frett done! I have implemented those changes.